src/templates/plugin.main.js

```diff
@@ -90,10 +90,22 @@ export default async (context) => {
     // Lazy-loading enabled
     if (lazy) {
       const { loadLanguageAsync } = require('./utils')
-
-      // Load fallback locale.
-      if (app.i18n.fallbackLocale && newLocale !== app.i18n.fallbackLocale) {
-        await loadLanguageAsync(context, app.i18n.fallbackLocale)
+      const i18nFallbackLocale = app.i18n.fallbackLocale
+
+      // Load fallback locale(s).
+      if (i18nFallbackLocale) {
+        if (typeof i18nFallbackLocale === 'object' && i18nFallbackLocale[newLocale]) {
+          let localesToLoadPromises = []
+          if (i18nFallbackLocale[newLocale]) {
```

Comment:
`i18nFallbackLocale[newLocale]` is already checked above so we can just map right away.

Reply:
Job's done